test/means/test_constant_mean.py

```diff
@@ -11,27 +11,33 @@
 
 
 class TestConstantMean(BaseMeanTestCase, unittest.TestCase):
-    def create_mean(self):
-        return ConstantMean()
+    batch_shape = None
+
+    def create_mean(self, prior=None):
+        return ConstantMean(prior=prior, batch_shape=torch.Size([]))
 
     def test_prior(self):
-        prior = NormalPrior(0.0, 1.0)
-        mean = ConstantMean(prior=prior)
+        if self.__class__.batch_shape is None:
```

Comment:
Any particular reason why you're accessing this via the class rather than via the instance (`self.batch_shape`)?

Reply:
I'm doing the same as what is done in the base class. It may not be necessary though, will check.